cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -154,6 +154,7 @@ public static Kafka createKafkaCluster(String clusterCmNamespace, String cluster
                     .endKafka()
                     .editZookeeper()
                         .withConfig(zooConfigurationJson)
+                        .withMetrics(metricsCm)
```

Comment:
There was a bug in tests. Value of `metricsEnabled` was set to true even if no metrics were set. With new client, the correct (false) value was set and therefore some tests were failing because they expected different values.